pawn/scripting/include/ripext/http.inc

```diff
@@ -53,6 +55,32 @@ enum HTTPStatus
 	HTTPStatus_HTTPVersionNotSupported = 505,
 };
 
+enum struct HTTPQueryParams
+{
+	int count;
+	char params[MAX_PATH_LENGTH];
+	
+	void Add(const char[] key, const char[] value)
+	{
+		if (this.count == 0)
+			Format(this.params, MAX_PATH_LENGTH, "%s=%s", key, value);
```

Comment:
The key and value need to be escaped with [curl_easy_escape](https://curl.haxx.se/libcurl/c/curl_easy_escape.html). Maybe we can add a static `HTTPClient.Escape` native.

Reply:
Can we escape it when start to sending request?🤔

Reply:
Adding escape on HTTPClient::BuildURL seen to be better

Reply:
You can't escape the path in `BuildURL`, as that would escape all `/`, `?`, `&` and `=` characters. You only want to escape special characters in keys and values. 

I'm also not a fan of a static Escape method (maybe Encode is better btw), but I believe that's the only way to do it without creating a methodmap.